docs/src/atomistic/outputs.rst

```diff
@@ -167,3 +167,43 @@ Energy ensemble gradients
 
 The gradient metadata for energy ensemble is the same as for the ``energy``
 output (see `Energy gradients`_).
+
+.. _features:
+
+Features
+^^^^^^^^
+
+Last layer features represent the outputs generated by the model before the final predictions,
+such as energy or forces, are made. They are associated with the ``"features"`` key in the model
+outputs, and must adhere to the following metadata:
+
+.. list-table:: Metadata for features output
+  :widths: 2 3 7
+  :header-rows: 1
+
+  * - Metadata
+    - Names
+    - Description
+
+  * - keys
+    - same as `Energy`_
+    - same as `Energy`_
+
+  * - samples
+    - same as `Energy`_
+    - same as `Energy`_
+
+  * - components
+    - same as `Energy`_
+    - same as `Energy`_
+
+  * - properties
+    -
+    - the features can have arbitrary properties.
+
+  * - gradients
+    -
+    - the features should not have any explicit gradients, all gradient calculations are done using autograd.
```

Comment:
I think we'll want to allow explicit gradients as well, just in case. I'll write this part.